Civi/Civioffice/EventSubscriber/ActivityCiviOfficeTokenSubscriber.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/*
+ * Copyright (C) 2024 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+declare(strict_types = 1);
+
+namespace Civi\Civioffice\EventSubscriber;
+
+use Civi\Api4\Activity;
+use Civi\Api4\ActivityContact;
+use Civi\Core\Event\GenericHookEvent;
+
+final class ActivityCiviOfficeTokenSubscriber extends AbstractCoreEntityCiviOfficeTokenSubscriber {
+
+  public function onCiviOfficeTokenContext(GenericHookEvent $event): void {
+    parent::onCiviOfficeTokenContext($event);
+
+    if ($this->getEntityType() === $event->entity_type) {
+      $activityContact = ActivityContact::get(FALSE)
```

Comment:
Let's add a code comment about what we're doing here, this used to be commented with:
```php
// Add implicit contact and case token context for activities.
```
… also applies to all other additions to `$event->context`

Reply:
Changed.